pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -125,6 +125,7 @@
 
   # must be lua51Packages
 , luaPackages
+, luajitPackages
```

Comment:
please dont add one. luaPackages is already overwritten to `luajitPackages` in the call (but we keep referring to the more generic "luaPackages" since neovim can also run with PUC lua, aka 5.1).

Reply:
I had to add this line since `luaPackages` would cause `nix build .#vimPlugins.image-nvim` to error with `Package ‘lua5.1-magick-1.6.0-1’ in /nix/store/9hxz2pwnyvmjmj5hw92wnvb2hnvnic4z-source/pkgs/development/lua-modules/generated-packages.nix:2422 is marked as broken, refusing to evaluate.` Don't know if there's another way to fix this

Reply:
luaPackages is used only for nvim-teal-maker (I added it it seems). Ideally we should use the neovim interpreter I guess, not sure how to do that properly yet so I will leave it for later.